### PR TITLE
Release 20210128

### PR DESCRIPTION
**Pull request by ElenaFdR**
This release represents the first major change to publiccode.net's design since before the Foundation launched. Thanks, everyone who contributed! 

## Review checklist template

- [ ] Correct Markdown and formatting
- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [ ] All content reviewed

-----
[View rendered careers/index.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/careers/index.md)
[View rendered codebases/index.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/codebases/index.md)
[View rendered codebases/openzaak.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/codebases/openzaak.md)
[View rendered codebases/signalen.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/codebases/signalen.md)
[View rendered membership/index.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/membership/index.md)
[View rendered team/ben-cerveny.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/team/ben-cerveny.md)
[View rendered team/deborah-meibergen.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210128/team/deborah-meibergen.md)

### CODE REVIEW

**Comment by angelplasma** [comment]
Release incorporates structural and content updates to the header navigation and footer, via https://github.com/publiccodenet/jekyll-theme

File: assets/style_codebases.scss [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+.cb-detail-intro {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  @include respond-to('small') {
+    flex-direction: column;
+  }
+}
+.cb-detail-info {
+  //border-top: 1px solid var(--fpc-grey);
```
Comment (bvhme) [inline assets/style_codebases.scss:289]: These commentted out rules seem to be for development and should probably be removed.

File: assets/style_codebases.scss [hunk 2]
```diff
@@ -0,0 +1,348 @@
+---
+---
+
+@import url('https://fonts.googleapis.com/css?family=Muli:300');
+
+:root {
+  --fpc-yellow: #FFF500;
```
Comment (bvhme) [inline assets/style_codebases.scss:7]: As a rule we don't abbreviate the name of our organization nor endorse specific fan developed acronyms. These color names can just be that, I suggest find and replacing to simply remove the `fpc-` and use the color names as variables. So `--yellow`.

File: assets/style_codebases.scss [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+    @warn "Unfortunately, no value could be retrieved from `#{$breakpoint}`. "
+      + "Available breakpoints are: #{map-keys($breakpoints)}.";
+  }
+}
+
+
+// LIVE CSS OVERRIDES - HOW BEST TO RECONCILE?
+
+p, h1, h2, h3, h4, h5, ul, pre, blockquote, .highlight {
+  max-width: none;
```
Comment (bvhme) [inline assets/style_codebases.scss:53]: Careful: this can cause blocks of text to have excessively long line length.

**Review by bvhme — APPROVED** [review]
Great work! I especially love the new codebase pages.

There is one thing I'd consider problematic, however I'm going to approve regardless to enable this great work to continue.

Problems:

* When printing the codebase page (in Firefox and Chrome) the links in the _Find out more_ section are truncated and thus unusable, while the URLs to links in the footer just don't show up at all (but that might be a theme problem).

Warnings:

* Some filenames are really hard to understand, like `signalen_g01.png`, this makes it hard for me to review and contribute to files where that is used as I need to open a file viewer to understand. I'd recommend filenames like `signalen-screenshot-1.png` or similar.
* I can't actually find this change but some paragraphs on the codebase overview pages are so wide that my eyes literally can't find the beginning of the next line and have to travel over 60em to the beginning of the next rule. I suggest adding a maximum width for text.
* The boxes on the codebase pages lists next to the screenshot can get incredibly narrow, to a one word per line point. I suggest enforcing a minimum width for text.
* Only one image is visible on the codebase pages, with no visual clue to whether other's are available.
* Whitespace in the stylesheet is a bit inconsistent

The rest of the inline comments are also warnings.

**Comment by angelplasma** [comment]
@bvhme This is an excellent list of notes! I'll spend more time with these over the weekend, and split off fixes and adjustments with response notes.

Thank you Boris :D

File: index.html [hunk 4]
```diff
@@ -9,8 +9,8 @@ <h1 class="title">We help public organizations collectively develop and maintain
 
     <p>This results in higher quality services for the public that are more cost effective, with less risk and more local control.</p>
 
-    <p>We define ‘public code’ as open source software developed by public organizations, together with the policy and guidance needed for reuse.</p>
-    <p><a href="https://about.publiccode.net/CONTRIBUTING.html">Join us</a>.</p>
+    <p class="subtext">We define ‘public code’ as open source software developed by public organizations, together with the policy and guidance needed for reuse.</p>
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply index.html:12]: Captured in #148

File: script/test.sh [hunk 5]
```diff
@@ -5,4 +5,10 @@ set -e # halt script on error
 bundle exec jekyll build
 
 # Check for broken links and missing alt tags, ignore edit links to GitHub as they might not exist yet
-bundle exec htmlproofer --url-ignore "/github.com/(.*)/edit/" ./_site
+
+bundle exec htmlproofer \
+    --url-ignore "/github.com/(.*)/edit/" \
+    --url-ignore "/github.com/,/twitter.com/" \
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply script/test.sh:11]: Captured as #150

File: assets/lightbox.css [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  right: 5%;
+}
+
+a.lb-btn-prev, a.lb-btn-next {
+  transition: all .15s ease-out;
+  cursor: pointer;
+}
+a.lb-btn-prev::after, a.lb-btn-next::after {
+  content:'';
+  background-image: url("data:image/svg+xml;utf-8,\
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply assets/lightbox.css:68]: Captured as #151.

File: assets/style_codebases.scss [hunk 7]
```diff
@@ -0,0 +1,348 @@
...
+  }
+  // If the key doesn't exist in the map
+  @else {
+    @warn "Unfortunately, no value could be retrieved from `#{$breakpoint}`. "
+      + "Available breakpoints are: #{map-keys($breakpoints)}.";
+  }
+}
+
+
+// LIVE CSS OVERRIDES - HOW BEST TO RECONCILE?
```
[the remark this answers is not in the document]
Comment (angelplasma) [reply assets/style_codebases.scss]: Was a todo. Now resolved.

File: assets/style_codebases.scss (hunk 1 above)
Comment (angelplasma) [reply assets/style_codebases.scss]: Agree. Removed.

File: assets/style_codebases.scss (hunk 2 above)
Comment (angelplasma) [reply assets/style_codebases.scss:7]: Captured in https://github.com/publiccodenet/publiccode.net/issues/152

File: assets/style_codebases.scss (hunk 3 above)
Comment (angelplasma) [reply assets/style_codebases.scss:51]: Captured in https://github.com/publiccodenet/publiccode.net/issues/153

**Pull request merged**